src/02-cta/style.scss

```diff
@@ -69,7 +69,7 @@ $mojblocks-color-white: #ffffff;
 
     .mojblocks-button, a.mojblocks-button {
         background-color: $mojblocks-color-yellow;
-        color: $mojblocks-color-black !important;
+        color: $mojblocks-color-black;
```

Comment:
`!important` may have been placed here to prevent a colour change when the button was clicked.